.github/workflows/semantic.yml

```diff
@@ -65,63 +42,19 @@ jobs:
       - name: Build warplib
         run: yarn warplib
 
+      - name: Setup gcc-11 Environment
+        run: |
+          sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+          sudo apt install -y gcc-11 gcc-11-multilib g++-11 g++-11-multilib
+          gcc-11 --version
+
       - name: Test code execution
         run: |
-          yarn testnet:benchmark &
-          node build/autoRunSemanticTests.js
+          yarn testnet:benchmark_semantic &
+          SM_BATCH=${{ matrix.SM_BATCH }} node build/autoRunSemanticTests.js
```

Comment:
There's `env` property that can be used to set env variables

Reply:
done!